packages/build-tools/src/builders/android.ts

```diff
@@ -109,8 +111,15 @@ async function buildAsync(ctx: BuildContext<Android.Job>): Promise<void> {
       });
 
       ctx.logger.info('Cache restored successfully');
-    } catch (err) {
-      ctx.logger.warn({ err }, 'Failed to restore cache');
+      await asyncResult(
+        spawnAsync('ccache', ['--zero-stats'], { env: ctx.env, logger: ctx.logger, stdio: 'pipe' })
+      );
+    } catch (err: any) {
+      if (err.response.status === 404) {
+        ctx.logger.warn('No cache found for this key, ensure it was created with eas/save_cache');
```

Comment:
I think we want to `info`.

Also, actually, maybe the action for users here is not to ensure, but rather to… just build haha (let's not ask them to ensure anything)

Reply:
We can also check what does github actions print in their cache action

Reply:
Though I think our message may be even less actionable than theirs given our cache integration is much more integrated and we can hopefully expect like 90% of users to just never disable cache and be happy with the results of what we're devising here, whereas in GHA it's all manual